### ISSUE TITLE
Custom logo

### ISSUE THREAD

**Issue by ironicbadger** [m1dnight/exbin#19]
Lots of issues from me! Here's another one. 

We're going to adopt Exbin as the pastebin for all listener feedback at Jupiter Broadcasting and we'd like to be able to set our own logo and title for the page and tabs if possible please.

**Comment by m1dnight** [comment]
Okay, happened to do it today. 

Here's what you do:

 - Update to the newest version (0.0.3). 
 - Assume your logo is called `my_logo.png`, and it's located on your host at `/containers/exbin/data/my_logo.png`. 
 - Add the environment variable `LOGO_FILENAME` to your instance and give it the value `my_logo.png`
 - Mount the logo as a volume in the Docker image like so: `- /containers/exbin/data/my_logo.png:/app/priv/static/images/my_logo.png`
 

I have pushed the newest version to Docker hub, and updated the docker-compose.yml file in the repo. 

I also added a `BRAND` variable that allows you to change the name, btw. 
 
 - Christophe
